src/fpylll/fplll/pruner.pyx

```diff
@@ -35,6 +35,10 @@ from fplll cimport svp_probability as svp_probability_c
 from fpylll.util import adjust_radius_to_gh_bound, precision, get_precision
 from fpylll.util cimport check_float_type, check_precision, check_descent_method, check_pruner_metric
 
+IF HAVE_LONG_DOUBLE:
+    from fplll import FT_LONG_DOUBLE
+    from decl import mpz_ld
```

Comment:
I noticed that this import isn't used explicitly, nor or some of the similar ones on line 25, but I kept it anyways in case there was something non-obvious I was missing.